lib/JsonClient.js

```diff
@@ -57,18 +60,22 @@ JsonClient.prototype.parse = function parse(req, callback) {
     function parseResponse(err, req2, res, data) {
         var obj;
         var resErr = err;
+        var parseErr;
 
         try {
             if (data && !/^\s*$/.test(data)) {
```

Comment:
Why do we avoid the parse error if it is an empty string?

Reply:
This is legacy: https://github.com/restify/node-restify/issues/203

I don't feel strongly about whether to continue supporting or not. Anyone else?

Reply:
Guess this is outside the scope of this PR then. Makes sense!